src/Components/Navbar/AdminNavbar.js

```diff
@@ -21,10 +21,10 @@ export default function AdminNavbar(props) {
   ];
 
   return (
-    <Navbar light className='admin-nav navbar-expand-md'>
+    <Navbar dark className='admin-nav navbar-expand-md'>
       <NavbarBrand href='/'>
-        Software & Computer Engineering Society
-      </NavbarBrand>
+        Admin Page     
```

Comment:
Do we wanna call the Admin side `Admin Page` now? Its more of a collection of pages